tests/caasp/rebootmgr.pm

```diff
@@ -110,7 +110,7 @@ sub run {
     record_info 'Maint-window', 'Test maint-window strategy';
     check_strategy_maint_window;
 
-    if (!check_var('SYSTEM_ROLE', 'microos')) {
+    if (!(check_var('SYSTEM_ROLE', 'microos') || check_var('SYSTEM_ROLE', 'kubeadm'))) {
```

Comment:
```suggestion
    if (check_var('SYSTEM_ROLE', 'microos') || is_caasp('caasp')) {
```

Reply:
We don't want this to run on microos, microOS doesn't have etcd either